ui/app/routes/vault/cluster.js

```diff
@@ -58,6 +59,7 @@ export default Route.extend(ModelBoundaryRoute, ClusterRoute, {
     const id = this.getClusterId(params);
     if (id) {
       this.get('auth').setCluster(id);
+      this.get('permissions').getPaths.perform();
```

Comment:
I think there's a codemod that should change all of these for us app wide (to do later, not in this PR!), but we should also probably develop some style guidelines around `this.get` vs `this.item` and maybe component usage and a few other things. Just making this note to remind myself to do that.